docs/README.md

```diff
@@ -1,17 +1,121 @@
-<!--
-title: IBC Go Documentation
-footer:
-  newsletter: false
--->
+TODO: UPDATE
 
-# IBC
+# IBC-Go Documentation
 
-Welcome to the IBC-Go documentation!
+Welcome to the IBC-Go documentation! This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
 
-The Inter-Blockchain Communication protocol (IBC) is an end-to-end, connection-oriented, stateful protocol for reliable, ordered, and authenticated communication between heterogeneous blockchains arranged in an unknown and dynamic topology.
+## Translating
 
-IBC is a protocol that allows blockchains to talk to each other.
+TODO: UPDATE or REMOVE
 
-The protocol realizes this interoperability by specifying a set of data structures, abstractions, and semantics that can be implemented by any distributed ledger that satisfies a small set of requirements.
+## Docs Build Workflow
 
-IBC can be used to build a wide range of cross-chain applications that include token transfers, atomic swaps, multi-chain smart contracts (with or without mutually comprehensible VMs), and data and code sharding of various kinds.
+The documentation for IBC-Go is hosted at <https://ibc.cosmos.network>.
+
+built from the files in this (`/docs`) directory for
+[main](https://github.com/cosmos/ibc-go/tree/main/docs).
+
+## docusaurus.config.js
+
+Docusaurus configuration file is located at `./docusaurus.config.js`. This file contains the configuration for the sidebar, navbar, footer, and other settings. Sidebars are created in `./sidebars.js`.
+
+## Links
+
+**NOTE:** Strongly consider the existing links - both within this directory
+and to the website docs - when moving or deleting files.
+
+Relative links should be used nearly everywhere, having discovered and weighed the following:
+
+### Relative
+
+Where is the other file, relative to the current one?
+
+- works both on GitHub and for the VuePress build
+- confusing / annoying to have things like: `../../../../myfile.md`
+- requires more updates when files are re-shuffled
+
+### Absolute
+
+Where is the other file, given the root of the repo?
+
+- works on GitHub, doesn't work for the VuePress build
```

Comment:
ditto

Reply:
I removed this section and replaced it with an issue tracker